server/index.js

```diff
@@ -1,131 +1,21 @@
 const fileUpload = require('express-fileupload')
 const express = require('express')
 const path = require('path')
-const fs = require('fs')
-const rimraf = require('rimraf')
 
 const app = express()
 
-const { isFileAttached } = require('./utils/middlewares')
-const { getAllFiles } = require('./utils/files.js')
-
 // default options
 app.use(fileUpload())
 
 app.use(express.static(path.join(__dirname, '..', '/client/build')))
 
-const defaultLocation = '..'
-
-// Get list of all files
-app.get('/files', (req, res) => {
-	return getAllFiles(`${defaultLocation}/snippets`).then(files =>
-		res.send({ files })
-	)
-})
+// Routers
+const labelRouter = require('./routes/label');
+const snippetRouter = require('./routes/snippet');
 
-app.get('/file', (req, res) => {
-	const { folder, file } = req.query
-	const filePath = `${defaultLocation}/snippets/${folder}/${file}`
-	fs.readFile(filePath, (err, data) => {
-		if (err)
-			return res.status(404).send({
-				success: false,
-				message: `File ${file} doesn't exist!`
-			})
-		return res.status(200).send({ success: true, file: data.toString() })
-	})
-})
+app.use(labelRouter);
+app.use(snippetRouter);
```

Comment:
`app.use('/snippet',snippet)`
And do the same with snippet, this is to keep a standard for when new routes are created for like bookmarks or something.

Reply:
Yes, sounds good to me. Do you want it to be something like `app.use('/api/snippet',snippet)` ?
Or just `/snippet` ?